Mathlib/Algebra/Module/LinearMap/Basic.lean

```diff
@@ -143,6 +142,12 @@ abbrev LinearMapClass (F : Type*) (R : outParam (Type*)) (M M₂ : Type*)
   SemilinearMapClass F (RingHom.id R) M M₂
 #align linear_map_class LinearMapClass
 
+-- @[simp]
```

Comment:
?

Reply:
It gives a lint error:
#check @LinearMapClass.map_smul /- simp can prove this:
  by simp only [@map_smul]